sphinx/search/__init__.py

```diff
@@ -197,6 +208,9 @@ def dispatch_visit(self, node):
             self.found_words.extend(self.lang.split(node.astext()))
         elif issubclass(nodetype, title):
             self.found_title_words.extend(self.lang.split(node.astext()))
+        elif self.is_meta_keywords(node, nodetype):
+                keywords = re.search(r'content\=\"(.*?)\"', str(node)).group(1)
+                self.found_words.extend(self.lang.split(keywords))
```

Comment:
In the examples on docuitls docs, the keywords are separated with commas.
```
.. meta::
   :description: The reStructuredText plaintext markup language
   :keywords: plaintext, markup language
```
http://docutils.sourceforge.net/docs/ref/rst/directives.html#meta

I think splitting with comma is better.